.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -21,11 +21,11 @@ assignees: ''
 <!-- A clear and concise description of what you expected to happen. -->
 
 **Environment**
- - TensorRT-RTX version: 
```

Comment:
I think this change is OK but its subjective if a trailing space is preferred. Let me know if we should revert this and I can make the change.